sdks/python/apache_beam/yaml/docs/README.md

```diff
@@ -0,0 +1,295 @@
+# Beam YAML Developer Documentation
```

Comment:
I think this level of documentation (for developers) is best kept as a living doc (which you already have) rather than checked into the repo.